packages/cubejs-cubestore-driver/src/WebSocketConnection.ts

```diff
@@ -21,13 +21,15 @@ export class WebSocketConnection {
       webSocket.readyPromise = new Promise<WebSocket>((resolve, reject) => {
         webSocket.lastHeartBeat = new Date();
         const pingInterval = setInterval(() => {
-          if (webSocket.readyState !== WebSocket.CLOSED) {
+          if (webSocket.readyState === WebSocket.OPEN) {
```

Comment:
I think we should ping only when the connection is opened.

![image](https://user-images.githubusercontent.com/572096/116285929-57362a80-a797-11eb-8697-7bd14e9165e9.png)